ios/2_build_toolchain.sh

```diff
@@ -77,31 +87,38 @@ function install_lib_sdl2() {
 	)
 }
 
-install_lib_icu_native
-
-set_build_flags
-
-install_lib_zlib
-install_lib $LIBPNG_DIR $LIBPNG_ARGS
-install_lib $FREETYPE_DIR $FREETYPE_ARGS --without-harfbuzz
-install_lib $HARFBUZZ_DIR $HARFBUZZ_ARGS
-install_lib $FREETYPE_DIR $FREETYPE_ARGS --with-harfbuzz
-install_lib $PIXMAN_DIR $PIXMAN_ARGS
-install_lib_cmake $EXPAT_DIR $EXPAT_ARGS
-install_lib $LIBOGG_DIR $LIBOGG_ARGS
-install_lib $LIBVORBIS_DIR $LIBVORBIS_ARGS
-install_lib_mpg123
-install_lib $LIBSNDFILE_DIR $LIBSNDFILE_ARGS
-install_lib_cmake $LIBXMP_LITE_DIR $LIBXMP_LITE_ARGS
-install_lib $SPEEXDSP_DIR $SPEEXDSP_ARGS
-install_lib_cmake $WILDMIDI_DIR $WILDMIDI_ARGS
-install_lib $OPUS_DIR $OPUS_ARGS
-install_lib $OPUSFILE_DIR $OPUSFILE_ARGS
-install_lib_cmake $FLUIDLITE_DIR $FLUIDLITE_ARGS -DENABLE_SF3=ON
-install_lib_meson $INIH_DIR $INIH_ARGS
-install_lib $LHASA_DIR $LHASA_ARGS
-install_lib_cmake $FMT_DIR $FMT_ARGS
-install_lib_icu_cross
-install_lib_liblcf
-
-install_lib_sdl2
+function build() {
+	install_lib_icu_native
```

Comment:
Please move this icu native build out (like macOS does it). The native ICU must be built for the platform you are building on (macOS), not the target (iOS).